packages/alert/src/close-icon.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import {Button} from '@smashing/button'
+
+const CloseButton = styled(Button)`
+  margin-left: auto;
+  box-sizing: content-box;
+  height: auto;
+  svg {
+    color: ${_ => _.theme.colors.text.muted};
+  }
+  &:hover {
+    background: transparent;
```

Comment:
Default background is ok, don't remove it

Reply:
Left unset background